crates/polars-time/src/chunkedarray/rolling_window/dispatch.rs

```diff
@@ -82,6 +82,7 @@ where
         return Ok(Series::new_empty(ca.name(), ca.dtype()));
     }
     let ca = ca.rechunk();
+    let by = by.rechunk();
```

Comment:
The error comes from L112
```rust
    let by_values = by.cont_slice().map_err(|_| {
        polars_err!(
            ComputeError:
            "`by` column should not have null values in 'rolling by' expression"
        )
    })?;
```

The values already get re-chunked, so the `by` ones probably should too?